controllers/authentication/openidconnect_controller.go

```diff
@@ -60,15 +60,17 @@ func (r *OpenIDConnectReconciler) Reconcile(ctx context.Context, req ctrl.Reques
 	err := r.Get(ctx, req.NamespacedName, config)
 	if err != nil {
 		if apierrors.IsNotFound(err) {
+			log.Info("not found - removing OIDC authenticator handler from store")
 
 			r.deleteHandler(req.Name)
 
 			return reconcile.Result{}, nil
 		}
+		return reconcile.Result{RequeueAfter: 10 * time.Second}, err
```

Comment:
If you want to make use of the exponential back off, I think you should do 
```suggestion
		return reconcile.Result{}, err
```

Now, it will be retried every 10 seconds which is not nice and can put excessive load on the system.